styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -394,7 +390,7 @@ public void create(Environment environment) {
     startScheduler(scheduler, executor);
     startRuntimeConfigUpdate(styxConfig, executor, dequeueRateLimiter);
     startCleaner(cleaner, executor);
-    startConcurrentWorkflowInstanceIndexing(storage, executor);
+
```

Comment:
remove empty line?

Reply:
More or less intentionally. Next statement is not startSomething. :D

Reply:
ok =)